include/cantera/thermo/DebyeHuckel.h

```diff
@@ -643,6 +643,7 @@ class DebyeHuckel : public MolalityVPSSTP
      *       independent variable however.
      *
      * @param rho Input density (kg/m^3).
+     * @deprecated To be removed after Cantera 2.5.
```

Comment:
True. I also changed the `Phase::density` method as it is not overloaded anywhere.

Reply:
I don't think we should make the method non-virtual without first marking it as deprecated (it's not the right term, since the method will still be there, but we can clarify that in the comment), as that would break any user code which currently relies on that being a virtual method.

I think there's probably a path to removing `PureFluidPhase::setDensity`, but we don't have to pursue that now.

Reply:
Regarding making a virtual method non-virtual: I don’t see a good way of notifying deprecation other than temporarily renaming the function?

Reply:
@speth ... I am reversing making `Phase::density` non-virtual. It's not necessary and I don't see a good way of letting users know.

Regarding `PureFluidPhase::setDensity` there probably is a way, but I'm not sure that this is necessary at the moment.